### PR TITLE
Fix photon binary resolution

### PR DESCRIPTION
**Pull request by timsuchanek**
We renamed the binary of photon from `prisma` to `query-engine-${PLATFORM}`.
In order to support that, I changed the special case to copy the whole runtime directory.

### CODE REVIEW

File: test/unit/prisma-photon/actual.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  "test/unit/prisma-photon/input.js",
+  "test/unit/prisma-photon/node_modules/@generated/photon/index.js"
+]
```
Comment (styfle) [inline test/unit/prisma-photon/actual.js]: This file shouldn't be committed

File: test/unit/prisma-photon/node_modules/@generated/photon/runtime/prisma [hunk 2]
```diff
@@ -1 +0,0 @@
-asdf
```
Comment (styfle) [inline test/unit/prisma-photon/node_modules/@generated/photon/runtime/prisma]: Perhaps you should rename this file from `prisma` to `query-engine-darwin`

File: test/unit/prisma-photon/actual.js (hunk 1 above)
Comment (timsuchanek) [reply test/unit/prisma-photon/actual.js]: Thanks! `actual.js` should then probably be added to `.gitignore`.

File: test/unit/prisma-photon/node_modules/@generated/photon/runtime/prisma (hunk 2 above)
Comment (timsuchanek) [reply test/unit/prisma-photon/node_modules/@generated/photon/runtime/prisma]: Yep, as node_modules are gitignored, I had to force add it.

**Pull request merged**